src/components/repository-table/repository-table.tsx

```diff
@@ -195,77 +220,82 @@ export const RepositoryTable = ({
 
   return (
     <GridItem>
+      <StyledHeading>Segments</StyledHeading>
       <StyledTable>
-        <Grid>
-          <GridItem xs={6}>
-            <LeftTableRow>
-              <label>
-                <Checkbox
-                  checked={globalCheckbox}
-                  onChange={() => handleGlobalCheckboxClick()}
-                />
-              </label>
-              <TableEntry>Total:</TableEntry>
-            </LeftTableRow>
-          </GridItem>
-          <GridItem xs={6}>
-            <TableValue>{Math.round(total * 100) / 100} $</TableValue>
-          </GridItem>
-        </Grid>
         {costPerRepository.map((repository, index) => {
           if (checkedRepositories.length === costPerRepository.length) {
             return (
-              <Grid key={index}>
-                <GridItem xs={6}>
-                  <LeftTableRow>
-                    <label>
-                      <Checkbox
-                        checked={checkedRepositories[index].checked}
-                        onChange={() => {
-                          let isOneCheckBoxInactive = false;
-                          setCheckedRepositories(
-                            checkedRepositories.map((item, position) => {
-                              let isItemChecked;
-                              if (index === position) {
-                                isItemChecked = !item.checked;
-                              } else {
-                                isItemChecked = item.checked;
-                              }
-                              if (!isItemChecked) {
-                                isOneCheckBoxInactive = true;
-                              }
-                              return {
-                                checked: isItemChecked,
-                                repositoryName: item.repositoryName,
-                              };
-                            })
-                          );
-                          if (isOneCheckBoxInactive) {
-                            setGlobalCheckbox(false);
-                          }
-                        }}
-                      />
-                    </label>
-                    <TableEntry>
-                      <ColorIcon
-                        repositoryName={repository.repositoryName}
-                        colorContext={colorsPerRepositoryName}
-                      >
-                        ⬤
-                      </ColorIcon>
-                      {repository.repositoryName}
-                    </TableEntry>
-                  </LeftTableRow>
-                </GridItem>
-                <GridItem xs={6}>
-                  <TableValue>{`${
-                    Math.round(repository.totalCost * 100) / 100
-                  } $`}</TableValue>
-                </GridItem>
-              </Grid>
+              <Separator key={index}>
```

Comment:
Using the `index` as key is usually a bad idea. Better use something unique from the array item itself. Here it could be `repository.name`.